config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```
@@ -203,3 +203,57 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure-dualstack
+    decorate: true
+    decoration_config:
+      timeout: 2h
+    always_run: false
+    optional: true
+    path_alias: k8s.io/kubernetes
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201005-0b243c0-master
+        command:
+        - runner.sh
+        - kubetest
+        args:
+        # Generic e2e test args
+        - --test
+        - --up
+        - --down
+        - --build=quick
+        - --dump=$(ARTIFACTS)
+        # Azure-specific test args
+        - --provider=skeleton
+        - --deployment=aksengine
+        - --aksengine-agentpoolcount=2
+        - --aksengine-admin-username=azureuser
+        - --aksengine-creds=$(AZURE_CREDENTIALS)
+        - --aksengine-orchestratorRelease=1.19
+        - --aksengine-mastervmsize=Standard_DS2_v2
+        - --aksengine-agentvmsize=Standard_D4s_v3
+        - --aksengine-deploy-custom-k8s
+        - --aksengine-location=westus2
+        - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
+        - --aksengine-template-url=https://gist.githubusercontent.com/aramase/21c3bdb44509c0f70f71389903d49c67/raw/1a9957626edc2b5c7913c741b9487fde123d0570/dualstack-prow.json
+        - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+        # Specific test args
+        - --test_args=--ginkgo.focus=\[Feature:IPv6DualStackAlphaFeature\]|\[\Conformance\]
```

if we don't want to match on the square brackets is ok too, but is better to have them, also there is a typo because you are escaping the C in conformance
Actually, I would match in everything Feature dual stack, so we don't have to keep modifying this job
```suggestion
        - --test_args=--ginkgo.focus=\[Feature:IPv6DualStack.*\]|\[Conformance\]
```

---

the \[ escape the square bracket and match on them

---

disregard this, the tests have two tags, but all of them have this one

---

@aramase the conformance tag is wrong though, it should be `\[Conformance\]` instead of `\[\Conformance\]`

---

@aojea Updated!